s32/mcux/devices/S32K146/S32K146_device.h

```diff
@@ -786,4 +786,30 @@ typedef struct {
  * @}
  */ /* end of group RTC_Peripheral_Access_Layer */
 
+/* ----------------------------------------------------------------------------
+   -- ADC Peripheral Access Layer
+   ---------------------------------------------------------------------------- */
+
+/*!
+ * @addtogroup ADC_Peripheral_Access_Layer ADC Peripheral Access Layer
+ * @{
+ */
+
+/* ADC - Peripheral instance base addresses */
+/** Peripheral ADC0 base address */
+#define ADC0_BASE                                (0x4003B000u)
```

Comment:
let'se use the values from `s32/drivers/s32k1/BaseNXP/header/S32K146_ADC.h`, e.g. `#define ADC0_BASE IP_ADC0_BASE`. See the rest of the definitions in this file as a reference.

Reply:
ok

Reply:
no, keep the definitions in s32/mcux/devices/S32K146/S32K146_device.h as in your previous commit, but use instead the values from the S32K146_ADC.h header, e.g. see as a reference https://github.com/zephyrproject-rtos/hal_nxp/blob/master/s32/mcux/devices/S32K146/S32K146_device.h#L563-L573